css/snackbar.css

```diff
@@ -0,0 +1,12 @@
+.snackbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-bottom: 5rem;
+  width: 40rem;
+  background: var(--charcoal-black);
+  color: var(--font-color-special);
+  padding: 2rem 0rem;
+  font-size: 1.5rem;
+  margin: 1rem auto;
+}
```

Comment:
Suggestion: can add a hover effect on 'RETRY' and 'X' like `cursor: pointer`, since they will become clickable later on.

Reply:
ok

Reply:
done